dumpling/dumpling.go

```diff
@@ -274,21 +274,18 @@ func (m *Dumpling) constructArgs() (*export.Config, error) {
 	return dumpConfig, nil
 }
 
-// detectAnsiQuotes tries to detect ANSI_QUOTES from upstream. If success, change EnableANSIQuotes in subtask config
-func (m *Dumpling) detectAnsiQuotes() {
+// detectSQLMode tries to detect SQL mode from upstream. If success, write it to LoaderConfig
+func (m *Dumpling) detectSQLMode() {
 	db, err := sql.Open("mysql", m.dumpConfig.GetDSN(""))
 	if err != nil {
 		return
 	}
 	defer db.Close()
-	enable, err := utils.HasAnsiQuotesMode(db)
+
+	sqlMode, err := utils.GetGlobalVariable(db, "sql_mode")
```

Comment:
Session here?

Reply:
dumpling didn't expose it's connection, so in fact we can't see its session variable. and by `GetDSN` and config, we can change its session variable different than global ones